### DISCUSSION TITLE
Link to SPORTV

### DISCUSSION THREAD

**Discussion by Enrickyb** [iptv-org/community#607] [Q&A] [answered]
I didn't find any sportv or premiere channels.

**Comment by BellezaEmporium** [comment]
DMCAd. (as far as I remember)

**Reply by Enrickyb** [reply]
What is DMCAd?

**Reply by BellezaEmporium** [reply, marked as answer]
A DMCA copyright notice has been issued concerning a certain set of channels streaming specific tournaments. SPORTV is one of them.
